### PR TITLE
Feature/je 7 wyswietlenie numerow telefonow uzytych w tresci maila

### PR DESCRIPTION
**Pull request by Cinek1812**
Hi guys and one girl,
Added new functionality, please check :)
if ok Merge 
If only our contentment will operate, then we add this function to main class JBDemail :)

### CODE REVIEW

File: src/main/java/com/jbd/DisplayPhoneNumbers.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    List<String> phoneList = new ArrayList<>();
+                    phoneList.add(phoneNumber.trim());
+                    resultMap.put(email.getFrom(), phoneList);
+                    LOGGER.info("Added record to map: " + resultMap.get(email.getFrom()) + " for user : " + email.getFrom());
+                    } else {
+                    if (resultMap.containsKey(email.getFrom())) {
+                        LOGGER.info("Found User: " + email.getFrom());
+                        isDuplicate = searchDuplicates(resultMap, phoneNumber);
+                        if (!isDuplicate) {
+                            LOGGER.info("No dupliacets, adding number: " + phoneNumber +" for: " +email.getFrom());
```
Comment (kkuczynska) [inline src/main/java/com/jbd/DisplayPhoneNumbers.java]: literówka w LOGGER.info

File: src/main/java/com/jbd/DisplayPhoneNumbers.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                    }
+
+                }
+
+            }
+
+
+        }
+
+        LOGGER.info("Reuslts: " + resultMap.toString());
```
Comment (kkuczynska) [inline src/main/java/com/jbd/DisplayPhoneNumbers.java]: literówka w LOGGER.info

File: src/test/java/com/jbd/DisplayPhoneNumbersTest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.*;
+
+/**
```
Comment (kkuczynska) [inline src/test/java/com/jbd/DisplayPhoneNumbersTest.java]: komentarz

**Comment by Cinek1812** [comment]
Fixed bugs

**Pull request merged**